guides.md

```diff
@@ -771,11 +773,226 @@ When a project begins, generate a [Summary Report](https://clockify.me/reports/s
 
 1. Timesheet PDFs sent to clients or for internal payroll calculation should be archived in our [shared drive](https://link.hypha.coop/drive) under the `Timesheets` directory
 
+## Virtual Machines
```

Comment:
This is missing some key information about how our infra is set up. Including that we have a jump server and that we need to be on the VPN to access it. Please include this information somewhere:

1. Connect to VPN
* you'll need a config file from Hypha Infra
* your password will be shared with you on [Hypha's passbolt](https://pass.hypha.coop)

2. Connect to Ansible 1 (our jump server)
* `ssh sysadmin@ansible1.hypha.prod -p 9154 -i ~/.ssh/id_rsa`
* you'll need to ensure that your key has been added. You can do that by making a PR [here](https://github.com/hyphacoop/configurations/tree/master/hyphacoop/ssh-public-keys).